src/Microsoft.CodeAnalysis.Razor.Workspaces/DefaultTagHelperDescriptorFactory.cs

```diff
@@ -180,13 +217,21 @@ private IEnumerable<string> GetAllowedChildren(INamedTypeSymbol type, ErrorSink
 
             foreach (var name in allowedChildren)
             {
-                var valid = TryValidateName(
+                if (string.IsNullOrWhiteSpace(name))
```

Comment:
These changes (and a few other minor ones) were added after the prototype was done.